test/google_crawler/search_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleCrawler.SearchTest do
+  use GoogleCrawler.DataCase, async: true
+  alias GoogleCrawler.Search
+  alias GoogleCrawler.Search.Keyword
+
+  describe "list_keywords/2" do
+    test "lists the keywords for given user" do
+      user = insert(:user)
+      %{id: keyword_id} = insert(:keyword, name: "travel", user: user)
```

Comment:
Should we insert another keyword that does not belong to user to test that it really return the correct keywords.